app/assets/stylesheets/common/base/topic-post.scss

```diff
@@ -1500,8 +1500,10 @@ span.mention {
 }
 
 #topic-footer-buttons {
+  position: relative;
   margin: var(--below-topic-margin) 0;
   padding: 0;
+  z-index: z("base"); // places above proceeding relative parent
 
```

Comment:
Relative positioning is needed to change child element to parent element scope, z("base") = 1, places over proceeding element.

![CleanShot 2023-12-15 at 12 26 30](https://github.com/discourse/discourse/assets/69276978/ccf1a699-c3ee-4815-9f73-b3b0cf3d14a4)